examples/sample-input-byo.tfvars

```diff
@@ -6,7 +6,6 @@
 # These required variables' values MUST be provided by the User
 prefix                                  = "<prefix-value>"
 location                                = "<aws-location-value>" # e.g., "us-east-1"
-ssh_public_key                          = "~/.ssh/id_rsa.pub"
```

Comment:
The more I think about the removal of this the more I believe we should keep it surfaced. This is a pretty big item as it's a security item. I know the default is also the same value so the functionality does not change, but not having this in everyone's face, I believe hide this important item. Thoughts or comments?

Reply:
good point. We could just move it from the Required section to the main body in the example files.

One other thing to consider: they ssh key is not currently optional for the nfs vm (although, for security reasons, one could argue that only the jump VM really needs it). But that would be a different PR. So far, no one has requested it.